python-threatexchange/threatexchange/cli/dataset_cmd.py

```diff
@@ -96,58 +110,67 @@ def init_argparse(cls, settings: CLISettings, ap: ArgumentParser) -> None:
         )
```

Comment:
question about existing command: only-tags takes a sequence? can you add some help text for how to delineate each?

Reply:
This flag may not actually work, will remove if so

Reply:
Looks like it is in use, but has never worked.

setting nargs=+ generates the default help text